commands/proposal_submission.go

```diff
@@ -36,27 +36,13 @@ func checkProposalSubmission(cmd *commandspb.ProposalSubmission) Errors {
 		if cmd.Rationale != nil {
 			if len(strings.Trim(cmd.Rationale.Description, " \n\r\t")) == 0 {
 				errs.AddForProperty("proposal_submission.rationale.description", ErrIsRequired)
-			} else if len(cmd.Rationale.Description) > 1024 {
-				errs.AddForProperty("proposal_submission.rationale.description", ErrMustNotExceed1024Chars)
+			} else if len(cmd.Rationale.Description) > 20000 {
```

Comment:
GQL schema comment does not reflect this change

Reply:
```
type ProposalRationale {
  """
  Description to show a short title / something in case the link goes offline.
  This is to be between 0 and 1024 unicode characters.
  This is mandatory for all proposals.
  ```

Reply:
fixed